pkg/scheduler/framework/plugins/nodevolumelimits/csi_manager.go

```diff
@@ -0,0 +1,41 @@
+package nodevolumelimits
+
+import (
+	storagev1 "k8s.io/api/storage/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	storagelisters "k8s.io/client-go/listers/storage/v1"
+	fwk "k8s.io/kube-scheduler/framework"
+)
+
+// CSIManagerImpl is a implementation of the CSIManager interface.
+type CSIManagerImpl struct {
```

Comment:
I'd prefer to follow other code naming pattern in the scheduler pkg and rename it to `CSIManager` or `DefaultCSIManager`